pool/db_test.go

```
@@ -57,6 +254,30 @@ func testInitDB(t *testing.T) {
 		}
 		return nil
 	})
+
+	// Persist the pool mode.
+	err = db.Update(func(tx *bolt.Tx) error {
```

This error is never checked